Modix.Services/Core/MessageLogBehavior.cs

```diff
@@ -1,29 +1,36 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Linq;
+using System.Security.Cryptography;
 using System.Text;
 using System.Threading.Tasks;
 using Discord;
 using Discord.WebSocket;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Modix.Data;
 using Modix.Data.Models.Core;
 using Modix.Services.Utilities;
-using Serilog;
 
 namespace Modix.Services.Core
 {
     public class MessageLogBehavior : BehaviorBase
     {
         private readonly DiscordSocketClient _discordClient;
 
-        public MessageLogBehavior(DiscordSocketClient discordClient, IServiceProvider serviceProvider) : base(serviceProvider)
+        public MessageLogBehavior(DiscordSocketClient discordClient, IServiceProvider serviceProvider, ILogger<MessageLogBehavior> logger) : base(serviceProvider)
         {
             _discordClient = discordClient;
+
+            Log = logger ?? NullLogger<MessageLogBehavior>.Instance;
```

Comment:
I thought we used Serilog for logging, which utilizes a `static` class for its API?

Does Serilog support the .NET Core Logging infrastructure? Are you just changing from static coupling to dependency injection? If so, this really belongs in a separate PR.

Reply:
Yeah, the injected logger logs to Serilog. [`UserInfoModule`](https://github.com/discord-csharp/MODiX/blob/6489654dd0f02fdd474b6c10ac5da5a54dc0515b/Modix/Modules/UserInfoModule.cs#L17) does the same.